docs/en/observability/slo-create.asciidoc

```diff
@@ -18,6 +18,12 @@ From here, complete the following steps:
 . <<set-slo>>.
 . <<slo-describe>>.
 
+[NOTE]
+====
+For SLOs to function, the cluster must include one or more nodes with both `ingest` and `transform` {ref}/modules-node.html#node-roles[roles] (they can co-exist or be distributed across separate nodes).
```

Comment:
```suggestion
For SLOs to function, the cluster must have one or more nodes with both `ingest` and `transform` {ref}/modules-node.html#node-roles[roles]. The roles can exist on the same node or be distributed across separate nodes.
```

Reply:
Sorry missed one...I've tweaked the text in the other statement to match this one.